Source/WebKit/UIProcess/WebAuthentication/Mock/MockHidService.cpp

```diff
@@ -55,7 +55,19 @@ void MockHidService::platformStartDiscovery()
 
 Ref<HidConnection> MockHidService::createHidConnection(IOHIDDeviceRef device) const
 {
-    return MockHidConnection::create(device, m_configuration);
+    auto connection = MockHidConnection::create(device, m_configuration);
```

Comment:
nit - smart pointer guidelines say this should explicitly use Ref
```suggestion
    Ref connection = MockHidConnection::create(device, m_configuration);
```